dts/msm8929-samsung-r04.dts

```diff
@@ -18,4 +18,19 @@
 			i2c-address = <0x25>;
 		};
 	};
+
+	on7lte-chn {
+		model = "Samsung Galaxy On7 2015 (SM-G6000)";
+		compatible = "samsung,on7lte-chn", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "G6000*";
+
+		// The stock Samsung bootloader seems to pick <0x0C01FF01 4> as the board id as supposed to <0xCE08FF01 3>
```

Comment:
Does "as supposed to" exist or did you mean

```suggestion
		// The stock Samsung bootloader seems to pick <0x0C01FF01 4> as the board id as opposed to <0xCE08FF01 3>
```

hehe, oppo...

Reply:
You're right, my mistake. Guess I heard it somewhere when I was younger and never got rid of it.